tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/config/fallback/ConfigPropertyGlobalVsMethodTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.fault.tolerance.tck.config.fallback;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+
+/**
+ * Test class for fallback configuration
+ * @author carlosdlr
+ */
+public class ConfigPropertyGlobalVsMethodTest extends Arquillian {
+
+    @Inject
+    private BeanWithFallback bean;
+
+    @Deployment
+    public static WebArchive deployAnotherApp() {
+        JavaArchive testJar = ShrinkWrap
+            .create(JavaArchive.class, "ftConfigFallback.jar")
+            .addClasses(BeanWithFallback.class)
+            .addAsManifestResource(
+            new StringAsset(
```

Comment:
Use the `ConfigAnnotationAsset`?

Reply:
@Ladicek sorry but I don't understand this comment, can you explain? thank you

Reply:
I understand now what you mean, you refer to the ConfigAnnotationAsset class to build the properties file.

Reply:
I was checking the class ConfigAnnotationAsset and looks like that is designed for handling time values, scale timeouts and enable annotations. I don't see in the class a  way to set this kind of values `fallbackMethod=fallbackGlobalConfForServiceA`.

Reply:
Something like:

```java
new ConfigAnnotationAsset()
    .set(BeanWithFallback.class, "serviceA", Fallback.class, "fallbackMethod", "fallbackGlobalConfForServiceA")
    .set(BeanWithFallback.class, "serviceA", Fallback.class, "applyOn", RuntimeException.class.getName())
```

(Actually just noticed your `applyOn=RuntimeException.class` -- that wrong AFAIK? The code above should fix that.)

Reply:
yes that will work thank you @Ladicek